src/rulesets/LinearAlgebra/norm.jl

```diff
@@ -37,35 +34,50 @@ function rrule(
                 _normInf_back(x, y, Δy)
             else
                 _normp_back_x(x, p, y, Δy)
-            end
-        end
+            end)
+            # , # in-place versions -- can be fixed when actually useful?
+            # dx -> if isempty(x) || p == 0
+            #     dx
+            # elseif p == 2
+            #     _norm2_back!(dx, x, y, Δy)
+            # elseif p == 1
+            #     _norm1_back!(dx, x, y, Δy)
+            # elseif p == Inf
+            #     dx .+= _normInf_back(x, y, Δy)  # not really in-place! could perhaps be improved
+            # elseif p == -Inf
+            #     dx .+= _normInf_back(x, y, Δy)
+            # else
+            #     dx .+= _normp_back_x(x, p, y, Δy)
+            # end
+            # )
```

Comment:
before we merge can we strip these commented out inplace sections from the PR.
And then open a issue about adding an inplace version of it, and link in that issue to the commit before the inplace functionality was deleted?